bokeh/tile_providers.py

```diff
@@ -14,6 +14,17 @@
     )
 )
 
+#: Provide toner background tiles from http://tile.stamen.com
+STAMEN_TONER_BACKGROUND = WMTSTileSource(
+    url='http://tile.stamen.com/toner-background/{Z}/{X}/{Y}.png',
```

Comment:
yes that is another service which is already included in `tile_providers.py`.  this is version which does not have labels baked in.